Lesson2/src/Homework2.java

```diff
@@ -1,20 +1,23 @@
+import java.io.Console;
+import java.util.Scanner;
+
 public class Homework2 {
 
     public static void main(String[] args) {
         System.out.println(sum(10, 20));
 //        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
         System.out.println(max(50, 49));
         System.out.println(average(new int[]{1, 2, 3, 4, 5}));
-//        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
-//        System.out.println(calculateHypotenuse());
+        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+        System.out.println(calculateHypotenuse());
     }
 
     /**
      * 1. Method should return sum of "a" and "b"
      * 2. (optional) if sum of a and b is more then int can store -- return -1;
      **/
-    public static int sum(int a, int b) {
-
+    public static int sum(int a, int b)
```

Comment:
please undo all changes like this. Open curly bracket should be on the same line with the method declaration